addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/NetatmoBindingConstants.java

```diff
@@ -52,10 +52,10 @@
     public static final String CHANNEL_LOCATION = "Location";
     public static final String CHANNEL_BOILER_ON = "BoilerOn";
     public static final String CHANNEL_BOILER_OFF = "BoilerOff";
-    public static final String CHANNEL_DATE_MAX_TEMP = "date_max_temp";
```

Comment:
Just a final check where these unused, or do the following four lines break backwards compatibility?